### PR TITLE
Updates for two October releases

### LINKED ISSUE #1039: [RELEASE] Updates for three releases in mid-October
## Server [RELEASE.2023-10-14T05-17-22Z](https://github.com/minio/minio/releases/tag/RELEASE.2023-10-14T05-17-22Z)

- [x] Adds new counter events ([PR #18232](https://github.com/minio/minio/pull/18232))

## MC [RELEASE.2023-10-14T01-57-03Z](https://github.com/minio/mc/releases/tag/RELEASE.2023-10-14T01-57-03Z)

- [x] Adds new flags to `mc admin trace` ([PR #4669](https://github.com/minio/mc/pull/4669))

## Operator [v5.0.10](https://github.com/minio/operator/releases/tag/v5.0.10)

- [x] Add buckets by modifying tenant YAML ([PR #1783](https://github.com/minio/operator/pull/1783))
- [x] Looks for `tls.crt` and `tls.key` files ([PR #1800](https://github.com/minio/operator/pull/1800))

### LINKED ISSUE #1045: [RELEASE] mc RELEASE.2023-10-24T05-18-28Z
mc [RELEASE.2023-10-24T05-18-28Z](https://github.com/minio/mc/releases/tag/RELEASE.2023-10-24T05-18-28Z) has changes that impact the docs:

- [x] New filter prevents duplicate transfers during `mv` and `cp` ([PR #4719](https://github.com/minio/mc/pull/4719))
- [x] mc flags can now have an ENV equivalent ([PR #4720](https://github.com/minio/mc/pull/4720))
- [x] `mc admin trace --stats` now shows min/max time ([PR #4723](https://github.com/minio/mc/pull/4723))
- [x] `mc admin trace --stats` now includes average time to first byte info ([PR #4726](https://github.com/minio/mc/pull/4726))
- [x] `mc mirror` now has a `--retry` flag ([PR #4715](https://github.com/minio/mc/pull/4715))
- [x] `mc admin replicate update` now has a `--bucket-bandwidth` flag ([PR #4687](https://github.com/minio/mc/pull/4687))

### PR DESCRIPTION
**Pull request by djwfyi**
- Adds additional flag for `mc admin trace`
- Adds info about supported secret types for k8s

Closes #1039 

Staged:
- http://192.241.195.202:9000/staging/server-2023-10-14/k8s/operations/network-encryption.html
- http://192.241.195.202:9000/staging/server-2023-10-14/k8s/operations/install-deploy-manage/deploy-minio-tenant.html#create-tenant-security-section
- http://192.241.195.202:9000/staging/server-2023-10-14/linux/reference/minio-mc-admin/mc-admin-trace.html#mc.admin.trace.-stats

For the `mc admin trace --stats` flag, also adding example output.

Closes #1045

### CODE REVIEW

File: source/operations/install-deploy-manage/deploy-minio-tenant.rst
Comment (feorlen) [inline source/operations/install-deploy-manage/deploy-minio-tenant.rst:570]: Are _opaque_ and _tls_ here keywords, of a sort that might get `inline code` ?

Comment (djwfyi) [reply source/operations/install-deploy-manage/deploy-minio-tenant.rst:570]: They are specific types in the linked K8s docs list.

**Review by ravindk89 — APPROVED** [review]
LGTM mod clarity on how much we can imply about support prior to 5.0.10

File: source/operations/install-deploy-manage/deploy-minio-tenant.rst
Comment (cesnietor) [reply source/operations/install-deploy-manage/deploy-minio-tenant.rst:580]: cc @cniackz

Comment (ravindk89) [reply source/operations/install-deploy-manage/deploy-minio-tenant.rst:580]: Sorry Cesar! Stupid github autoselect.

**Review by feorlen — APPROVED** [review]
LGTM pending Eng input

File: source/operations/install-deploy-manage/deploy-minio-tenant.rst
Comment (cniackz) [reply source/operations/install-deploy-manage/deploy-minio-tenant.rst:580]: Yes, @ravindk89, currently, cert-manager can function as expected when we provide the specified name for the certificate. Additionally, there is a pending PR that, once merged, will prevent the unnecessary request for this secret to be mounted. Therefore, I anticipate no issues after version 5.0.10.

**Review by cniackz — APPROVED** [review]
I agree with @Ravind. We should simply mention 'support added' without specifying any particular version, as that information would likely become outdated soon. Apart from that, the PR appears satisfactory to me.

File: source/operations/install-deploy-manage/deploy-minio-tenant.rst
Comment (cniackz) [reply source/operations/install-deploy-manage/deploy-minio-tenant.rst:580]: Ok PR got merged already: https://github.com/minio/operator/pull/1847/files

**Pull request merged**
